packages/@okta/vuepress-site/docs/guides/migrate-to-okta/migrate-users-password-hooks/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Import Users with Inline Password Hooks
+---
+
+## Migration Program Plan
+
+A migration program utilizes Okta's Inline Password Hook feature to seamlessly migrate users as they authenticate. The broad strokes look like:
+
+1. Create all the users from the legacy system in Okta with a provider set to: `IMPORT`. **NOTE:** This can be done in bulk and does NOT require individual user credentials.
```

Comment:
```suggestion
1. Create all the Users from the legacy system in Okta with a provider set to: `IMPORT`. Note that this can be done in bulk and does not require individual user credentials.
```